docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -87,13 +94,25 @@ TIP: To learn more about the query, select the **Threat Intel** tab, then click
+If these fields aren't present, {es-sec} does not perform the query and threat intelligence for the alert is not displayed.
+
+TIP: To learn more about the query, select the **Threat Intel** tab, then click the **Inspect** button next to the threat you want to inspect.
+
+Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the Threat Matched Detected section under the Threat Threat Intel tab with additional threat match data.
 
 *Threat Intel*
 
-Shows the number of individual threats matching the alert in reverse chronological order, with the most recent threats at the top. Each threat is labelled with values from the `matched.field` and `matched.atomic` fields. All mapped fields are displayed for each matched threat.
+Shows the number of individual threats matching the alert. You can expand and collapse threat details by clicking the arrow icon at the end of the threat label. Each threat is labelled with values from the `matched.field` and `matched.atomic` fields and displays the threat intelligence provider. If the alert does not contain threat intelligence data, the *Threat Intel* tab displays a message that none is available.
+
+Matched threats are organized into several sections, described below. Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
+
+* *Threat Matched Detected*: This section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. If indicator threat matches are not discovered, the section displays a message that none are available.
+* *Enriched with Threat Intelligence*: This section shows indicator matches that {es-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query timeframe, which looks at the past 30 days by default. Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected timeframe, the section displays a message that none are available.
+* *Other*: This section shows threats that do not match any threat indicators or threat intelligence fields that {es-sec} queries the alert for. If these types of threat matches are not discovered on the alert, the section does not display.
+
+Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
```

Comment:
I think this information was present on line 107, so maybe it's not necessary to repeat it here.

Reply:
You're correct! Removing it now, thanks. :)